RFC/zh_CN/0012-darwinia-bridge-core-interoperation-in-chainrelay-enabled-blockchains.md

```diff
@@ -239,6 +239,46 @@ FlyClient[6]介绍了一种新的交易验证的轻客户端方案，可以支
 
 详细设计可参考[FlyClient: Super-Light Clients for Cryptocurrencies](https://eprint.iacr.org/2019/226)
 
+
+
+FlyClient提供了super light client的方案，但是Chain Relay需要在区块链内置合约(模块)或者用户合约中实现super light client，因此在设计方案上需要增加一些考虑点：
+
+- Asymptotically and on Demand Proof Summit and Challenge Time
+
+  ChainRelay提交块头的如果只是一个人，则无法确认是诚实或是作恶的，或者说即使是善意的，提交的头也可能由于自然分叉而无效。在轻钱包协议中一个重要假设是轻钱包可以和多个全节点连接获得数据，并且至少其中一个是诚实的。所以，在我们的实现中，只有一个人提交头类似与只有一条连接，不足以确保数据的有效性。因此在用户使用这个ChainRelay的最新区块头及相关默克尔证明之前，需要增加一个Challenge时间，也就是最新的区块头被提交后，处于质疑状态，需要有一个challenge时段，在此时间过后没有人提交反对块（符合块基本交易且拥有更高难度，但MMR中不包含质疑块）或者收到赞同块（难度更高，但是MMR包含质疑块），最新提交的区块才被确认。此时，用户将可以使用这个Chain Relay验证在此最新块之前发生的交易是否存在。
+
+  
+
+  为了防止攻击者攻击，在提交块时需要附带一定金额的押金，在块被信任后，押金返回并附带一定比例的手续费池金额作为奖励；如果块被否决，则押金扣除，扣除的押金一部分进入手续费池，一部分直接奖励否决块的提交人（否决块也需要被信任后才成为否决块）。
+
+  
+
+  补充一下，否决块或者信任快，必须与信任快相隔至少x个高度，x由不同来源链特性决定，可等同于一般的确认块数，以防止攻击者连续发送分叉链上的多个块。同时也保证诚实relayer有时间发送否决块。
```

Comment:
typo: 否决块或者信任“快”，必须与信任“快”

Reply:
I removed this part because it seems to be related with finalization rather than challange:

https://github.com/darwinia-network/rfcs/commit/054799c41372e18aea2eaabd78449d073df4a4c7